code/datums/ai/sanity/sanityloss_behaviors.dm

```diff
@@ -4,13 +4,13 @@
 				"You left me behind!",
 				"You are the problem, YOU!!",
 				"I'll destroy everything...",
-				"It will end quickly, so relax. I�ll free you from this prison we call flesh."
+				"It will end quickly, so relax. I'll free you from this prison we call flesh."
 				)
 
 /datum/ai_behavior/say_line/insanity_suicide
 	lines = list(
-				"It's all my fault. It�s my responsibility...",
-				"I can hear someone. It�s the sound of back home. I just can�t stop hearing it�",
+				"It's all my fault. It's my responsibility...",
+				"I can hear someone. It's the sound of back home. I just can't stop hearing it�",
```

Comment:
Undefined symbol..?

Reply:
The " ’ " were being shown as questionmarks in the code.  I will fix the ? real quick.